### PR TITLE
Add unit tests for `OneSpanSign` methods

### LINKED ISSUE #14: Add tests for existing methods
CI workflow will be added after resolving #6. We'll need to add tests to the existing methods

### PR DESCRIPTION
**Pull request by hawschiat**
#### Key changes

- [x] Added unit tests for the methods in the `OneSpanSign` class
- [x] Fixed eslint config for test files

### CODE REVIEW

File: tests/OneSpanSign.test.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      ${'stream'} | ${Readable.from('abc1234')} | ${(data: any) => expect.objectContaining({ source: data })}
+    `('sends the provided payload and $type as request body', async ({ data, expectMatcher }) => {
+      const payload: UploadDocumentRequestPayload = {
+        name: 'mock document',
+      };
+      const packageId = 'mockId';
+
+      await oneSpan.uploadDocument(packageId, payload, data);
+
+      // Had to compare the values this way instead of using a snapshot because the boundary value is different each time
```
Comment (francisco-jara) [inline tests/OneSpanSign.test.ts:51]: Just fyi, if you ever want to set a custom boundary on a FormData object you can do it this way:

```
const formData = new FormData()
formData._boundary = '--------------------------my-custom-boundary'

console.info(formData.getHeaders())  
//  { 'content-type': 'multipart/form-data; boundary=--------------------------my-custom-boundary' }
```
Of course that'd be a little weird to add a `customBoundary` parameter to your `uploadDocument` method just to be able to use snapshots.

Anyway... I thought this might be good for peeps to know.

**Review by francisco-jara — APPROVED** [review]

**Review by abbieschenk — APPROVED** [review]
